### PR TITLE
chore: added codecov with ci git action

### PR DESCRIPTION
**Pull request by liujuping**

### CODE REVIEW

**Comment by liujuping** [comment]
fixed: https://github.com/alibaba/lowcode-engine/issues/856

File: packages/engine/README-zh_CN.md [hunk 1]
```diff
@@ -12,7 +12,11 @@
 
 [![NPM version][npm-image]][npm-url] [![NPM downloads][download-image]][download-url]
 
- [![][issues-helper-image]][issues-helper-url] [![Issues need help][help-wanted-image]][help-wanted-url]
+[![][issues-helper-image]][issues-helper-url] [![Issues need help][help-wanted-image]][help-wanted-url]
+
```
Comment (LeoYuan) [inline packages/engine/README-zh_CN.md:16]: 统一成 [![][][]] 的格式吧~

Comment (liujuping) [reply packages/engine/README-zh_CN.md:16]: done

Comment (liujuping) [reply packages/engine/README-zh_CN.md:16]: done

**Pull request merged**